addons/crm_iap_lead_enrich/models/crm_lead.py

```diff
@@ -87,14 +83,17 @@ def iap_enrich(self, from_cron=False):
                             iap_response = self.env['iap.enrich.api']._request_enrich(lead_emails)
                         except iap_tools.InsufficientCreditError:
                             _logger.info('Sent batch %s enrich requests: failed because of credit', len(lead_emails))
-                            if not from_cron:
+                            if not from_cron and not self._context.get('import_file'):
```

Comment:
Looks unrelated to this commit.

Reply:
Hello @awa-odoo 

Actually, we don't want to show the Bus notification if a user is importing the file.
Because it may trigger this notification multiple times on the user screen.

Thanks

Reply:
Hmmm.

To double check but I'm pretty sure the CRON will NOT share the same context as the import process anyway.

Reply:
Hello @awa-odoo 

Yes, you are right. Cron trigger is not sharing context.